server.js

```diff
@@ -44,36 +54,170 @@ app.post('/api/events', (req, res) => {
 })
 
 // Delete the event with given eventID
 app.delete('/api/event/:eventID', (req, res) => {
-  console.log(req.params.eventID);
-  res.send();
+  console.log("Deleting event with ID = " + req.params.eventID);
+  const eventId = req.params.eventID;
+  const queryString = "DELETE FROM Event WHERE EventID = ?";
+
+  getConnection().query(queryString, [eventId], (err, results, fields) => {
+    if (err) {
+      console.log("Failed to delete event " + eventId);
+      res.sendStatus(500);
+      return;
+    }
+    console.log("Event deleted!");
+    res.sendStatus(200);
+    return;
+  });
+
 })
 
 // User with userID joins event with eventID. Create such
 // entry in Participate table
 app.put('/api/event/:eventID/user/:userID', (req, res) => {
-  console.log(req.params.eventID);
-  console.log(req.params.userID);
-  res.send();
+  console.log("Updating attendee list for event: " + req.params.eventID);
+  const userId = req.params.userID;
+  const eventId = req.params.eventID;
+  const statusArray = ['Interested', 'Registered', 'Paid'];
+  const randomNumber = Math.floor(Math.random() * statusArray.length);
+  const queryString = "INSERT INTO Participate (UserID, EventID, Status) VALUES (?, ?, ?)";
+
+  getConnection().query(queryString, [userId, eventId, statusArray[randomNumber]], (err, results, fields) => {
+    if (err) {
+      console.log("Failed to update attendee list");
+      res.sendStatus(500);
+      return;
+    }
+    console.log("Participation for event " + eventId + " has been updated!");
+    res.sendStatus(200);
+    return;
+  });
+
 })
 
 // Create a user with the given firstName, lastName, dateOfBirth,
 // gender, cardNumber, expiryDate, holderName and CVC
 app.post('/api/user', (req, res) => {
-  console.log(req.body);
-  res.send({ userID: 0 });
-})
+  console.log("Trying to create a new user...");
+  const firstName = req.body.firstName;
+  const lastName = req.body.lastName;
+  const dateOfBirth = req.body.dateOfBirth;
+  const gender = req.body.gender;
+  const cardNumber = req.body.cardNumber;
+  const expiryDate = req.body.expiryDate;
+  const holderName = req.body.holderName;
+  const cvc = req.body.CVC;
+  const queryStringUser = "INSERT INTO User (FirstName, LastName, DateOfBirth, Gender) VALUES (?, ?, ?, ?)";
+  const queryStringDateOfBirth = "INSERT INTO DoB (DateOfBirth, Age) VALUES (?, ?)";
+  const queryStringCard = "INSERT INTO CreditCard (CardNumber, ExpiryDate, HolderName, CVC) VALUES (?, ?, ?, ?)";
+
+  getConnection().query(queryStringUser, [firstName, lastName, dateOfBirth, gender], (err, results, fields) => {
+    if (err) {
+      console.log("Failed to insert new user: " + err);
+      res.sendStatus(500);
+      return;
+    }
+    console.log("Inserted a new user with id: " + results.insertId);
+  });
+
+  // TODO: properly insert age instead of using Math.random()
+  getConnection().query(queryStringDateOfBirth, [dateOfBirth, (Math.floor(Math.random() * 10) * 3)], (err, results, fields) => {
```

Comment:
You can use `d = new date(); y = d.getFullYear() - date(dateOfBirth).getFullYear();` to get the difference between the current year and the given dob.